README.md

```diff
@@ -23,6 +23,7 @@
 
 - E2EE: End-to-end encryption
 - EDU: Ephemeral Data Unit
+- ELS: Event List Summary
```

Comment:
These could do with more explanation. 

Like these are the summaries of state and hidden events in the timeline: ex. `maeedigaro and 2 others joined and left, ohcrapthing joined, richcord changed their name, TRASH changed their avatar, corecircuit247.. joined and changed their avatar, JoshAtticus sent a hidden message`